kuma/static/styles/components/payments/_page.scss

```diff
@@ -75,6 +74,20 @@
         }
     }
 
+    .lead-in {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        border-top: 1px solid $neutral-500;
+        border-bottom: 1px solid $neutral-500;
+        padding: 24px 10px 24px;
+        max-width: 670px;
+
+        p {
+            margin-bottom: 0;
+        }
+    }
+
```

Comment:
This is an unfortunate duplication for now. Once the new account settings goes live, this will be deleted.